libpromises/crypto.c

```diff
@@ -833,6 +841,8 @@ static void SetupOpenSSLThreadLocks(void)
 #ifndef __MINGW32__
     CRYPTO_set_id_callback((unsigned long (*)())ThreadId_callback);
 #endif
+    // This is a no-op macro for OpenSSL >= 1.1.0
+    // The callback function is not defined then
```

Comment:
*not **used** then*

Reply:
Both are correct, but I meant **defined**, as in, there is a guard around the definition as well.

Reply:
Clarified.